File: backend/workstation-booking-system/wbs-api/Models/Workstation.cs
namespace wbs_api.Models;

public class Workstation
{
    public int Id { get; set; }
    public string Code { get; set; } = string.Empty;
    public string Wing { get; set; } = string.Empty;
    public int XPosition { get; set; }
    public int YPosition { get; set; }
    public int Width { get; set; }
    public int Height { get; set; }
    public bool IsActive { get; set; } = true;
}

File: frontend/workstation-booking-app/src/models/Workstation.ts
export type Wing = "NORTH" | "SOUTH";

export interface Workstation {
  id: number;
  bookingId?: number;
  code: string;
  wing: Wing;
  xPosition: number;
  yPosition: number;
  width: number;
  height: number;
  isActive: boolean;
  userId?: string;
  userName?: string;
  status: string;
}
